src/TSTransformer/nodes/expressions/transformCallExpression.ts

```diff
@@ -147,7 +144,7 @@ export function transformCallExpressionInner(
 		]);
 	}
 
-	const expType = state.typeChecker.getNonOptionalType(state.getType(node.expression));
+	const expType = state.typeChecker.getNonNullableType(state.getType(node.expression));
```

Comment:
What does this fix/change?

Reply:
With one of the TS versions, optional types got stricter (in parameter declarations, for example), and there's now a distinction between undefined and a lack of the type at all. But for our purposes, we need to exclude the undefined case altogether.